Makefile

```diff
@@ -169,6 +175,8 @@ install-smoke-tests:
 # Example:
 # make mode=sandbox create-smoke-tests
 # make mode=live create-smoke-tests
+# make mode=sandbox update=false create-smoke-tests
+# make mode=live update=false create-smoke-tests
```

Comment:
allow skipping update-prescriptions in smoke tests if you want to push through prescriptions as they are, defaulted to update